File: backend-java/app/src/main/java/org/example/Request.java
package org.example;

public class Request {
	String method;
	String path;
	String body;
	String contentType;
	public Request() {
		this.method = "GET"; // Default method
		this.path = "/";
		this.body = "";
		this.contentType = "application/json"; // Default content type
	}
	public Request(String method, String path, String body, String contentType) {
		this.method = method;
		this.path = path;
		this.body = body;
		this.contentType = contentType;
	}
}

File: backend-scala/app/bin/main/org/example/Request.kt
package org.example;

class Request(path: String, method: String, contentType: String, body: String) {
	var path: String = "/";
	var method: String = "GET";
	var contentType: String =  "text/plain";
	var body: String = "";
	init {
		this.path = path
		this.method = method
		this.contentType = contentType
		this.body = body
	}
}